packages/tools/lib/monitor-postcss/index.js

```diff
@@ -0,0 +1,65 @@
+const chokidar = require("chokidar");
+const commandLineArgs = require("command-line-args");
+const psList = require("ps-list");
+const fkill = require("fkill");
+const { exec } = require("child_process");
+
+const options = commandLineArgs([
+	{ name: "srcFiles", type: String },
+	{ name: "packageName", type: String },
+]);
+const packageName = options.packageName;
+
+const restartPostcss = async () => {
+	let retryCount = 5;
+	let processes;
+
+	while (retryCount) {
+		try {
+			console.log("Getting processes list...");
+			processes = await psList();
+			retryCount = 0;
+		} catch (e) {
+			console.log("failed, retrying...");
+			retryCount--;
+		}
+	}
+
+	const process = processes.find(p => p.cmd.includes("postcss") && p.cmd.endsWith(`-w --packageName=${packageName}`));
+	if (process) {
+		await fkill(process.pid);
+
+		exec(`npx nps build.styles.components`, (err, stdout, stderr) => {
+			if (err) {
+				console.log(`Could not rebuild CSS for ${packageName}`);
+				console.log(err);
+			}
+
+			console.log(stderr);
+			console.log(stdout);
+		});
+
+		exec(`npx nps watch.styles.components`, (err, stdout, stderr) => {
```

Comment:
not sure if starting `npx` from `yarn` won't have additional problems. could we reuse the `nps` api with `require`?

Reply:
I didn't find any. That was my initial intention too. I could only find `nps-utils`.